computation.cpp

```diff
@@ -26,23 +26,24 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "timer.h"
 #include <chrono>
 
-Computation::Computation(InputParameters& params, Progress& progress)
-    : params(params)
-    , progress(progress)
-    , verbosity(params.verbosity)
+Computation::Computation(int vrbsty, Progress& progress)
```

Comment:
Please use real words, including the vowels.

Reply:
Hi Bryn, broadly that seems like a good suggestion, but in this case, how do you want it implemented?  The natural name for this argument is "verbosity," but that is already taken here.

Reply:
Parameter names and initializer names resolve as you would hope in initializer lists, so you can actually do verbosity(verbosity) and it does the right thing.